app/va/va_onsite/va_onsite_client.py

```diff
@@ -32,12 +32,13 @@ def post_onsite_notification(self, data: dict):
             response = requests.post(url=f'{ self.url_base }/v0/onsite_notifications',
                                      data=json.dumps(data),
                                      headers=self._build_header())
+
+            self.logger.info(f'onsite_notifications POST response: status_code={response.status_code}, ' +
+                             f'json={response.json()}')
+
         except Exception as e:
             self.logger.exception(f'Exception in post_onsite_notification: {e}')
```

Comment:
I realize you didn't change this, but the exception logger includes a stack trace.  Stating the source is superfluous.
```
self.logger.exception(e)
```

Reply:
updated